manage/users/index.rst

```diff
@@ -100,6 +100,25 @@ The ones that aren’t are described below.
 
       To add one, go to **Manage > Organizations** in the Admin Panel.
 
+🏤 Secondary Organizations
+   This option allows you to assign more organizations (which are not the users
+   primary) to users.
```

Comment:
This option allows you to assign more organizations, in addition to the user's primary organization.